app/brave_main_delegate.cc

```diff
@@ -237,6 +238,7 @@ bool BraveMainDelegate::BasicStartupComplete(int* exit_code) {
     offline_pages::kPrefetchingOfflinePagesFeature.name,
     translate::kTranslate.name,
 #endif
+    brave_shields::features::kBraveDomainBlock.name,
```

Comment:
app/brave_main_delegate_browsertest.cc is usually updated at the same time when we modifying this.